src/inc/readytorun.h

```diff
@@ -15,7 +15,7 @@
 
 #define READYTORUN_SIGNATURE 0x00525452 // 'RTR'
 
-#define READYTORUN_MAJOR_VERSION 0x0002
+#define READYTORUN_MAJOR_VERSION 0x0003
```

Comment:
Would this be a good opportunity to unify the hash code algorithms between CoreCLR and CoreRT?

We already have the necessary change in #19546 - AFAIK it was only blocked because it wasn't a good enough reason to bump the major format version.

Cc @trylek @nattress

Reply:
This is not resetting minimum supported version. Old R2R images are still going to work fine with the change.

Reply:
OK, I guess that means that we can only make a change like this once we decide there's a business justification to breaking backwards compat and that is not happening now.

Reply:
https://github.com/dotnet/coreclr/pull/22041#issuecomment-460867440 would be that kind of change.

FWIW, I am not sure whether the current CoreRT hashing algorithm is the ultimate right one. It does not look good that it requires UTF8->UTF16 conversion before hashing.